### PR TITLE
Adding Plausible Analytics.

### PR DESCRIPTION
**Pull request by thomascenni**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [X] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [X] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version (only for service mail with image: bytemark/smtp I am using latest tag because it's the only one)
- [X] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [X] Icon is added as a png file to the logos directory.

### CODE REVIEW

File: public/v4/apps/plausible.yml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            SECRET_KEY_BASE: $$cap_gen_random_hex(64)
+            SIGNING_SALT: $$cap_gen_random_hex(24)
+            DISABLE_REGISTRATION: "true"
+            DISABLE_SUBSCRIPTION: "true"
+            MAILER_EMAIL: $$cap_RELAY_USERNAME
+            SMTP_HOST_ADDR: srv-captain--$$cap_appname-mail
+            SMTP_HOST_PORT: "25"
+            GEOLITE2_COUNTRY_DB: "/geoip/GeoLite2-Country.mmdb"
+        volumes:
+            - $$cap_appname-geoip-data:/geoip:ro
```
Comment (githubsaturn) [inline public/v4/apps/plausible.yml:68]: drop `:ro` as it's not respected by CapRover

**Review by githubsaturn — COMMENTED** [review]
Thanks for submitting this, looks like a large application. Would you be able to add some more details to instructions.start?
- The minimum/recommended RAM and CPU
- Also, list out all the different images that is being installed since it's a collection of various images maintained by different parties.

File: public/v4/apps/plausible.yml (hunk 1 above)
Comment (thomascenni) [reply public/v4/apps/plausible.yml:68]: In their page:
https://docs.plausible.io/self-hosting
There are no minimum requirements.
I'll list the different images used in the description.

**Comment by githubsaturn** [comment]
Thanks!

**Comment by LittleBigFrog** [comment]
Hello, 

Unfamiliar with clickhouse. Hope it is the right place to ask.

Plausible on a 5$ digitalocean droplet uses 100% of CPU.
It seems to be linked to Clickhouse, thanks to investigation by @RealOrangeOne :

https://theorangeone.net/posts/calming-down-clickhouse/
Could we have clickhouse low ressource setup with default no logging via parameters?

Thank you

**Pull request merged**